source/components/reveal/HxReveal.html

```diff
@@ -1,6 +1,4 @@
 <button id="toggle" aria-expanded="false">
   <slot name="summary"></slot>
 </button>
-<div id="content">
```

Comment:
The `<div>` wrapper here introduces an unnecessary level of CSS inheritance. So I unwrapped the unnamed slot.